backend/src/main/java/com/example/backend/controller/AuthenticationController.java

```diff
@@ -19,33 +27,62 @@
 import lombok.RequiredArgsConstructor;
 
 @RestController
+@CrossOrigin
 @RequestMapping("/api/v1/auth")
 @RequiredArgsConstructor
 public class AuthenticationController {
     private final AuthenticationService authenticationService;
     private final JwtService jwtService;
 
+    private static final String clientURL = "${tsa.clientUrl}";
+
     @PostMapping("/register")
-    public ResponseEntity<AuthenticationResponse> register(@RequestBody RegisterRequest request) {
+    public ResponseEntity<?> register(@RequestBody RegisterRequest request) {
         AuthenticationResponse registerResponse = authenticationService.register(request);
 
-        return ResponseEntity.status(registerResponse.getStatus()).body(registerResponse);
+        return ResponseEntity.status(registerResponse.getStatus()).body(null);
```

Comment:
ce doresti sa faci aici?

Reply:
pai nu mai e nevoie sa trimitem nimin in response body la register

Reply:
Si dc nu returnam un obiect cu field urile null pentru consistenta?